documentation/src/components/page.ts

```diff
@@ -30,7 +30,7 @@ export class Page extends LayoutElement {
     }
 
     public resetScroll() {
-        if (!this.contentElement) return;
+        if (!this.contentElement || !this.contentElement.scroll) return;
 
         this.contentElement.scroll(0, 0);
```

Comment:
Not worth spending much thought on, but should we use `scrollTop` here instead for broader support for returning to the top of the page?